app/jobs/email_verification_code_job.rb

```diff
@@ -0,0 +1,11 @@
+class EmailVerificationCodeJob < ApplicationJob
+  retry_on Mailgun::CommunicationError
```

Comment:
can we retry on similar error for the text messaging job? I wonder if there's an equivalent communication error from twilio's end